File: TagMemo-java/app/src/main/java/com/timejh/tagmemo_java/model/Memo.java
package com.timejh.tagmemo_java.model;

import io.realm.RealmList;
import io.realm.RealmObject;

/**
 * Created by tokijh on 2017. 4. 30..
 */

public class Memo extends RealmObject {

    public String id;

    public String parentGroupId;
    public String title;
    public String content;
    public RealmList<HashTag> tags;

    public String last_date;
    public boolean isValidated;
}

File: TabMemo-Swift/TabMemo-Swift/Memo.swift
//
//  Memo.swift
//  TabMemo-Swift
//
//  Created by 윤중현 on 2017. 6. 1..
//  Copyright © 2017년 윤중현. All rights reserved.
//

import RealmSwift

class Memo: Object {
    dynamic var id: String = ""
    
    dynamic var parentGroupId: String = ""
    dynamic var title: String = ""
    dynamic var content: String = ""
    let tags = List<HashTag>()
    
    dynamic var last_date: String = ""
    dynamic var isValidated: Bool = false
}
